frontend/src/app/shared/components/grids/widgets/members/members.component.html

```diff
@@ -40,7 +40,7 @@
 </div>
 
 <div class="op-members-widget--grid--widget-footer">
-  <a *ngIf="membersAddable"
+  <a *ngIf="membersAddable$ | async"
```

Comment:
Guess this will fix https://community.openproject.org/projects/openproject/work_packages/48720